ui-app/client/src/components/Video/Video.tsx

```diff
@@ -97,6 +97,15 @@ function Video(props: ComponentProps) {
 	const [fullScreenState, setFullScreenState] = useState<String>('expand');
 	const [isFirstTimePlay, setFirstTimePlay] = useState<boolean>(true);
 
+	
+	const videoSrc = getSrcUrl(src);
+
+	useEffect(() => {
+        if (video.current && videoSrc) {
+            video.current.src = videoSrc;
+        }
+    }, [videoSrc]);
+
 	//videoRef
```

Comment:
Why is this required?

Reply:
By using this iam updating specific parts of a component , While using only key it will also work but it forces a re-render entire component so i used this to ensure unnecessary  unmounting and re-mounting the component.